components/service/ServiceCard.tsx

```diff
@@ -1,15 +1,26 @@
+'use client';
```

Comment:
I don't believe "use client" is necessary here... Apologies if I am overlooking something.

Reply:
<img width="368" alt="image" src="https://github.com/AlexDjangoX/portfolio-alexander/assets/79525229/34b2d61b-c0ed-4a65-94e7-5203ef9f973f">